pages/login.vue

```diff
@@ -1,10 +1,14 @@
 <template>
   <v-card>
     <v-form>
+      <div align="center">
+        <h1 class="mb-md-5" id="page-title">{{ $t("login.adminLogin") }}</h1>
+      </div>
       <div class="d-flex flex-row justify-content-center">
```

Comment:
would recommend moving these classes into sass as well and using an id/class that's relevenant to the actually element. like `div class="login-fields"`

Reply:
- added `class="login-fields"` to <h1> 
- removed the unnecessary `<div>`
- added style to `.login-fields`

Reply:
you actually can just use either the `id` or the `class` here. No need for both :)

Reply:
removed the `id`